package.json

```diff
@@ -46,19 +47,25 @@
     "url": "git://github.com/mozilla/shield-studies-addon-utils.git"
   },
   "scripts": {
-    "build": "cd webExtensionApis/study && webpack",
+    "build": "cd webExtensionApis/study && webpack  ",
     "docformat": "doctoc --title '**Contents**' docs/*.md && prettier '**/*.md' --write",
     "eslint": "eslint . --ext jsm --ext js --ext json",
     "eslint-fix": "npm run eslint -- --fix",
+    "fakeApi": "cd webExtensionApis/study && yaml2json schema.yaml -p > schema.json && node ../../bin/schemaToInterface.js ./schema.json > fakeApi.js",
```

Comment:
The `yaml2json schema.yaml -p > schema.json` should be a part of the ordinary `build` step in order to make schema.yaml the source of truth for the schema declaration. If it is not the source of truth and only specific to fakeApi generation, we should not overwrite schema.json since it is the file that gets bundled into the example add-ons.